packages/sdks/react-sdk/src/types.ts

```diff
@@ -128,6 +128,16 @@ export type DescopeProps = {
   // use to override client context in flow execution
   client?: Record<string, any>;
   styleId?: string;
+  onScreenUpdate?: (
+    screenName: string,
```

Comment:
why do we not exposing screen id as well?

Reply:
Because this is something the descoper cannot edit, 
We wanted to avoid a scenario where replacing a screen in a flow breaks the application code (because the id is different)